clusters/build-clusters/02_cluster/installation_params.md

```diff
@@ -0,0 +1,60 @@
+# Build02: installation parameters
+
+## [Configuring a GCP project](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html)
+
+I will ask team-dpp for help on this, as we did this for ci-infra on aws for `build01`.
+
+Use project "openshift-ci-infra" as we do for `api.ci`.
```

Comment:
`api.ci` is retiring. So we could reuse the project, right?

Reply:
How had is it to use a new one? I would not want us to mess something up here while api.ci is around.

Reply:
I can not answer that question for sure but I think changing the project shouldn't be much of an issue

Reply:
I will ask DPP for this.
James did it for the new project of build01.